src/components/select-box/select-box.njk

```diff
@@ -0,0 +1,43 @@
+{% from "select-box/macro.njk" import govukSelectBox %}
+
+
+{{ govukSelectBox(
+  id='select-box-1',
```

Comment:
i can make them consistent with the rest of components, but I set default empty here
```
{% macro govukSelectBox(classes=''
```
so that I dont need to specify classes when calling the macro. can change so that we show that classes is an available parameter